packages/application-extension/src/index.tsx

```diff
@@ -689,6 +691,31 @@ const mainCommands: JupyterFrontEndPlugin<void> = {
           // - by listening to this command execution.
         }
       });
+      commands.addCommand('application:move-tab', {
+        label: args => {
+          const dir = args?.['direction'] as string | undefined;
+          return dir
+            ? trans.__(`Move Tab ${dir[0].toUpperCase() + dir.slice(1)}`)
```

Comment:
This will not work with translations, we need to manually hard-code four strings for the four directions.

Either "Move Tab Left", "Move Tab Right" etc, or "Move Tab %1", and then nested `trans.__("Left")`, `trans.__("Right")` etc